src/Services/ongInfoServices.js

```diff
@@ -0,0 +1,15 @@
+import axios from "axios";
```

Comment:
this should be on slidesService, what you are returning on this endpoint is slides with images so, on slides service you should add a method to return getImagesSlides or something like that

Reply:
changes added